packages/shared/src/dto/workflows/workflow-response-dto.ts

```diff
@@ -1,4 +1,4 @@
-import { PreferencesResponseDto, StepResponseDto, WorkflowCommonsFields } from './workflow-commons-fields';
+import { PreferencesResponseDto, StepResponseDto, WorkflowCommonsFields } from './workflow-response.dto';
```

Comment:
@tatarco I think we made the exact parallel change in two PRs. Let's chat tomorrow to see how can we merge, the `workflow-response.dto`or `workflow-response-dto`.